lib/shared/store.js

```diff
@@ -157,7 +157,7 @@ const getAvailableDrives = (state) => {
 const storeReducer = (state = DEFAULT_STATE, action) => {
   switch (action.type) {
     case ACTIONS.SET_AVAILABLE_DRIVES: {
-      // action.data : Array<DriveObject>
+      // Type: action.data : Array<DriveObject>
```

Comment:
Is this some 'convention' I'm not aware of? If not, I think having two colons on the one line looks a bit strange?

Reply:
`Type:` isn't really a convention AFAIK, it's just a nod to the user reading it, but I can see why it would confuse and I'm not sure how to resolve this one now. `Type(action.data : Array<DriveObject>)`? `Type - action.data : Array<DriveObject>`?

Reply:
Anything wrong with just `action.data : Array<DriveObject>` - should be fairly self-evident what it means?